etc/config/c++.amazon.properties

```
@@ -3506,6 +3506,15 @@ libs.gemmlowp.versions=trunk
 libs.gemmlowp.versions.trunk.version=trunk
 libs.gemmlowp.versions.trunk.path=/opt/compiler-explorer/libs/gemmlowp/trunk
 
+libs.glaze.name=glaze
+libs.glaze.description=JSON, reflection, and interface library
+libs.glaze.versions=trunk:198
+libs.glaze.url=https://github.com/stephenberry/glaze
+libs.glaze.versions.trunk.version=trunk
+libs.glaze.versions.trunk.path=/opt/compiler-explorer/libs/glaze/trunk
+libs.glaze.versions.198.version=1.9.8
+libs.glaze.versions.198.path=/opt/compiler-explorer/libs/glaze/1.9.8
```

seems this needs to be v1.9.8; will fix and merge :)
```suggestion
libs.glaze.versions.198.path=/opt/compiler-explorer/libs/glaze/v1.9.8
```